data/content/assets/icon-removed-bookmark.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M15.845 6.06297C15.7824 5.87657 15.671 5.71036 15.5224 5.58162C15.3738 5.45288 15.1934 5.36633 15 5.33097L13.875 5.13097L12.146 6.85397L13.018 7.00997L10.568 9.64497L11.068 13.217L8.00001 11.618L6.70901 12.291L3.24101 15.759C3.39937 15.8593 3.58154 15.9156 3.76886 15.9223C3.95619 15.929 4.14191 15.8857 4.30701 15.797L8.00001 13.874L11.688 15.795C11.8671 15.8888 12.0686 15.9315 12.2703 15.9183C12.472 15.9051 12.6662 15.8365 12.8316 15.7202C12.9969 15.6038 13.127 15.4441 13.2075 15.2587C13.288 15.0733 13.3159 14.8692 13.288 14.669L12.679 10.311L15.605 7.16397C15.7406 7.01915 15.8344 6.84036 15.8767 6.64654C15.9189 6.45271 15.908 6.25108 15.845 6.06297ZM14.707 1.29297C14.5195 1.1055 14.2652 1.00018 14 1.00018C13.7348 1.00018 13.4805 1.1055 13.293 1.29297L10.6 3.98297L8.98401 0.73297C8.89342 0.549544 8.75317 0.395223 8.57922 0.287562C8.40526 0.179901 8.20458 0.123221 8.00001 0.12397C7.79544 0.123846 7.59488 0.180773 7.42088 0.288354C7.24688 0.395935 7.10632 0.549907 7.01501 0.73297L5.08901 4.59997L1.00701 5.32897C0.812091 5.36333 0.630054 5.44964 0.480089 5.57881C0.330125 5.70797 0.217788 5.87521 0.154923 6.06288C0.0920581 6.25056 0.0809936 6.45171 0.122896 6.64515C0.164798 6.83859 0.258115 7.01714 0.393009 7.16197L3.32001 10.31L3.16501 11.421L1.29301 13.293C1.1975 13.3852 1.12132 13.4956 1.06891 13.6176C1.0165 13.7396 0.988912 13.8708 0.987759 14.0036C0.986605 14.1363 1.01191 14.268 1.06219 14.3909C1.11247 14.5138 1.18672 14.6255 1.28061 14.7194C1.37451 14.8133 1.48616 14.8875 1.60905 14.9378C1.73195 14.9881 1.86363 15.0134 1.99641 15.0122C2.12919 15.0111 2.26041 14.9835 2.38241 14.9311C2.50442 14.8787 2.61476 14.8025 2.70701 14.707L14.707 2.70697C14.8945 2.51944 14.9998 2.26513 14.9998 1.99997C14.9998 1.73481 14.8945 1.4805 14.707 1.29297ZM2.98101 7.00997L6.43001 6.39297L8.00001 3.24297L9.11101 5.47497L5.20001 9.39097L2.98101 7.00997Z" fill="#737373"/>
+</svg>
```

Comment:
running this with svgo will give optimized version of this svg @Mardak don't see svgo in our node_modules, is that a recent change?